pages/index.js

```diff
@@ -1,3 +1,34 @@
+import styled from "styled-components";
+import movieData from "../db.json";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  return (
+    <main>
```

Comment:
You already have a `main` element in your `Layout` component, so you should erase this one here.